integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/datacompaction/CarbonIndexFileMergeTestCase.scala

```diff
@@ -111,6 +111,32 @@ class CarbonIndexFileMergeTestCase
     checkAnswer(sql("""Select count(*) from nonindexmerge"""), rows)
   }
 
+  test("verify index file merge for SI") {
```

Comment:
This test  case looks like a duplicate case. Already present in CarbonIndexFileMergeTestCaseWithSI#Verify command of index merge

Reply:
removed